examples/python/depth_guided_stable_diffusion/README.md

```diff
@@ -1,25 +1,108 @@
-  <source media="(max-width: 1200px)" srcset="https://static.rerun.io/depth-guided-stable-diffusion/bea9bfaf33ebed4296f576d931c8c8e6fdd08a21/1200w.png">
+  <source media="(max-width: 480px)" srcset="https://static.rerun.io/depth_guided_stable_diffusion/a85516aba09f72649517891d767e15383ce7f4ea/480w.png">
+  <source media="(max-width: 768px)" srcset="https://static.rerun.io/depth_guided_stable_diffusion/a85516aba09f72649517891d767e15383ce7f4ea/768w.png">
+  <source media="(max-width: 1024px)" srcset="https://static.rerun.io/depth_guided_stable_diffusion/a85516aba09f72649517891d767e15383ce7f4ea/1024w.png">
+  <source media="(max-width: 1200px)" srcset="https://static.rerun.io/depth_guided_stable_diffusion/a85516aba09f72649517891d767e15383ce7f4ea/1200w.png">
+  <img src="https://static.rerun.io/depth_guided_stable_diffusion/a85516aba09f72649517891d767e15383ce7f4ea/full.png" alt="Depth-guided stable diffusion example">
 </picture>
 
-A more elaborate example running Depth Guided Stable Diffusion 2.0.
+Leverage [Depth Guided Stable Diffusion](https://github.com/Stability-AI/stablediffusion?tab=readme-ov-file#depth-conditional-stable-diffusion) to generate images with enhanced depth perception. This method integrates depth maps to guide the Stable Diffusion model, creating more visually compelling and contextually accurate images.
 
-For more info see [here](https://github.com/Stability-AI/stablediffusion).
+## Used Rerun Types
+[`Image`](https://www.rerun.io/docs/reference/types/archetypes/image), [`Tensor`](https://www.rerun.io/docs/reference/types/archetypes/tensor), [`DepthImage`](https://www.rerun.io/docs/reference/types/archetypes/depth_image), [`TextDocument`](https://www.rerun.io/docs/reference/types/archetypes/text_document)
 
+## Background
+Depth Guided Stable Diffusion enriches the image generation process by incorporating depth information, providing a unique way to control the spatial composition of generated images. This approach allows for more nuanced and layered creations, making it especially useful for scenes requiring a sense of three-dimensionality.
+
+# Logging and Visualizing with Rerun
+The visualizations in this example were created with the Rerun SDK, demonstrating the integration of depth information in the Stable Diffusion image generation process. Here is the code for generating the visualization in Rerun.
+
+## Prompt
+Visualizing the prompt and negative prompt
+```python
+rr.log("prompt/text", rr.TextDocument(prompt))
+rr.log("prompt/text_negative", rr.TextLog(negative_prompt))
+```
+
+## Text
+Visualizing the text input ids, the text attention mask and the unconditional input ids
+```python
+rr.log("prompt/text_input/ids", rr.Tensor(text_input_ids))
+rr.log("prompt/text_input/attention_mask", rr.Tensor(text_inputs.attention_mask))
+rr.log("prompt/uncond_input/ids", rr.Tensor(uncond_input.input_ids))
```

Comment:
outdated, I changed some of these to bar charts explicitly. This was somewhat broken in this demo.

Reply:
looks like I didn't change `uncond_input/ids` but it's also not in the blueprint. I probably should have changed it as well